public/app/features/serviceaccounts/ServiceAccountCreatePage.tsx

```diff
@@ -13,16 +13,22 @@ import { AccessControlAction, OrgRole, Role, ServiceAccountCreateApiResponse, Se
 
 import { getNavModel } from '../../core/selectors/navModel';
 import { StoreState } from '../../types';
+import { OrgRolePicker } from '../admin/OrgRolePicker';
 
-interface ServiceAccountCreatePageProps {
+export interface Props {
   navModel: NavModel;
 }
 
+const mapStateToProps = (state: StoreState) => ({
+  navModel: getNavModel(state.navIndex, 'serviceaccounts'),
+});
+
 const createServiceAccount = async (sa: ServiceAccountDTO) => getBackendSrv().post('/api/serviceaccounts/', sa);
+
 const updateServiceAccount = async (id: number, sa: ServiceAccountDTO) =>
   getBackendSrv().patch(`/api/serviceaccounts/${id}`, sa);
 
-const ServiceAccountCreatePage: React.FC<ServiceAccountCreatePageProps> = ({ navModel }) => {
+export const ServiceAccountCreatePageUnconnected = ({ navModel }: Props): JSX.Element => {
   const [roleOptions, setRoleOptions] = useState<Role[]>([]);
   const [builtinRoles, setBuiltinRoles] = useState<{ [key: string]: Role[] }>({});
```

Comment:
Getting a 404 from `/api/access-control/builtin-roles?targetOrgId=1` due to introduction of basic roles in the create Service Account Page.

Probably missing some if checks where this is populated like below

```
      if (
        contextSrv.accessControlBuiltInRoleAssignmentEnabled() &&
        contextSrv.licensedAccessControlEnabled() &&
        contextSrv.hasPermission(AccessControlAction.ActionBuiltinRolesList)
      ) {
        const builtInRoles = await fetchBuiltinRoles();
        dispatch(builtInRolesLoaded(builtInRoles));
      }
 ```